WordPress/Classes/ViewRelated/Post/PostEditor+BlogPicker.swift

```diff
@@ -28,10 +28,6 @@ extension PostEditor where Self: UIViewController {
             }
             self.recreatePostRevision(in: blog)
             self.mediaLibraryDataSource = MediaLibraryPickerDataSource(post: self.post)
-
```

Comment:
Out of curiousity, if you kept this here but added your changes, would it be able to also check if the new blog had xpost suggestions and thus show them if it did?

Reply:
I'd move these lines to [here](https://github.com/wordpress-mobile/WordPress-iOS/pull/15591/files#diff-beacfffc3f8a87edf94895da980a81e356f1b5aa7998f70791bc7baec9a11731R436-R439) where they will get triggered both when a user switches blogs from within the editor as well as when the user navigates to the editor (e.g. from the My Site screen).

So as far as I know, the move ensures that Xposts are always prefetched. Does that answer your question?

Reply:
Interesting, it does! That's why you only had the one call vs the original two in different places. Thanks!

Reply:
Thanks for checking 👍